doc/en/multitest.rst

```diff
@@ -94,6 +94,26 @@ A MultiTest instance can be constructed from the following parameters:
     def post_testcase(self, name, env, result):
         pass
 
+  :py:func:`@testcase <testplan.testing.multitest.suite.skip_if>` decorator can
+  be used to define the precondition to determine if a testcase needs to run.
+  It accepts one or several functions, if any of them returns True, then the
+  decorated testcase will not be executed.
+
```

Comment:
typo: @testcase --> @skip_if, suggested change:

:py:func:`@skip_if<testplan.testing.multitest.suite.skip_if>` decorator can be used to annotate a testcase. It take one or more predicates, and if any of them evaluated to True, then  the testcase will be skipped by MultiTest instead of being normally executed. The predicate’s signature must name the argument “testsuite” or a MethodSignatureMismatch will be raised.

Reply:
modified